### DISCUSSION TITLE
zstash using old lcrc#dtn_bebop Globus endpoint

### DISCUSSION THREAD

**Discussion by darincomeau** [E3SM-Project/zstash#317] [General]
I'm trying to use zstash to transfer data from NERSC HPSS to LCRC, but it is trying to use the lcrc#dtn_bebop Globus endpoint which expires Dec 1, rather than the new LCRC Improv DTN endpoint

**Comment by forsyth2** [comment]
Workaround for now: Set the new endpoint (https://app.globus.org/file-manager/collections/15288284-7006-4041-ba1a-6b52501e49f1/overview) in `~/.zstash.ini`:
```
[local]
globus_endpoint_uuid = 15288284-7006-4041-ba1a-6b52501e49f1
```

For the next Unified, we can change the default endpoints -- #318.

**Reply by xuezhengllnl** [reply]
I had the same issue and the setting works for me. Thanks!
